src/services/ContentService.js

```diff
@@ -1108,7 +1108,10 @@ define(["structures/ContentCreateStruct", "structures/ContentUpdateStruct", "str
                     callback(error, views);
                     return;
                 }
-
+                if ( viewCreateStruct.getCriteria() && Object.keys(viewCreateStruct.getCriteria()).length !== 0 ) {
```

Comment:
I'm a bit confused. Here you are deprecating the usage of Criteria but the getters and setters for Criteria are created in the `ViewCreateStruct`. Can you explain this approach to me?